dotnet/src/SemanticKernel/AI/OpenAI/Clients/OpenAIClientAbstract.cs

```diff
@@ -42,22 +48,26 @@ internal OpenAIClientAbstract(ILogger? log = null)
         this._httpClientHandler = new() { CheckCertificateRevocationList = true };
         this.HTTPClient = new HttpClient(this._httpClientHandler);
         this.HTTPClient.DefaultRequestHeaders.Add("User-Agent", HTTPUseragent);
+        // Setting the timeout to infinite.
+        // Will use cancellation token for timeout, giving users capability to set timeout per call.
```

Comment:
this seems very dangerous. 100 seconds is already a lot, why would one ask for more? 

If the kernel is deployed inside an Azure function, this assumption won't work. AzF have a max duration of 230secs, AWS lambdas 900 secs.